charts/osm/values.yaml

```diff
@@ -5,8 +5,8 @@ OpenServiceMesh:
   replicaCount: 1
   image:
     registry: openservicemesh
-    pullPolicy: Always
-    tag: latest
+    pullPolicy: IfNotPresent
+    tag: v0.2.0
   imagePullSecrets:
     - name: acr-creds
```

Comment:
Can you please remove the ACR secret?

Reply:
yea let me make that section more generic